### ISSUE TITLE
接受参数和返回值个数不同

### ISSUE THREAD

**Issue by Ding-Chian** [taojingcong/ccks2021FEE#6]
evnet_train.py中107行代码：logits, bert_emb_1,bert_emb_2 = model(input_ids, segment_id, attn_masks, mod='ned_emb')，在这里看model的返回值应该有三个呀，但是model的真实返回值却只有一个（return logits），且接受的参数里面没有mod呀，请问一下，这怎么解决？

**Comment by KarvinaD** [comment]
我也遇到這個問題了，請問你解決了嗎？我跑出來的代碼結果都是0

**Comment by KarvinaD** [comment]
> evnet_train.py中107行代码：logits, bert_emb_1,bert_emb_2 = model(input_ids, segment_id, attn_masks, mod='ned_emb')，在这里看model的返回值应该有三个呀，但是model的真实返回值却只有一个（return logits），且接受的参数里面没有mod呀，请问一下，这怎么解决？

我也遇到這個問題了，請問你解決了嗎？我跑出來的代碼結果都是0
